bin/gpu-trace.py

```diff
@@ -519,8 +519,11 @@ def __init__(self):
         self.gpuvis = GetBinary('gpuvis')
         self.user = RunCommand('logname').strip()
 
-    def OpenTrace(self, path):
-        RunCommand(["sudo", "-u", self.user, self.gpuvis, path], True)
+    def OpenTrace(self, path, perf_path=None):
+        gpuvis_args = ["sudo", "-u", self.user, self.gpuvis, path]
+        if perf_path is not None:
+            gpuvis_args.append(perf_path)
+        RunCommand(gpuvis_args, True)
```

Comment:
I think this new signature for `OpenTrace()` is used in the previous commit. Could we get this block moved back in history so that history remains bisect-able? 

Not that we use do much bisects in this project, but if we can avoid it I'd rather have it available :)

Reply:
Absolutely.  I'm glad you caught this and I'm sorry I didn't.  My intention was for each of these commits to be atomic.

Reply:
Actually, open closer inspection this doesn't seem to be the case.  I've moved "Respect --no-gpuvis in standalone mode" earlier in history, with my other bugfixes, to make that more obvious.